usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -208,7 +211,10 @@ def generate_federal_account_query(queryset, account_type, tas_id):
         ),
         "agency_identifier_name": get_agency_name_annotation(tas_id, "agency_id"),
         "submission_period": FiscalYearAndQuarter("reporting_period_end"),
-        "last_modified_date" + NAMING_CONFLICT_DISCRIMINATOR: Max("submission__certified_date"),
+        "last_modified_date"
+        + NAMING_CONFLICT_DISCRIMINATOR: Cast(
+            Max(Coalesce("submission__certified_date", "submission__published_date")), output_field=DateField()
+        ),
```

Comment:
same as above
> Similar to comment in `load_multiple_submissions.py`, i'm not sure if we should consider updating of certified date alone as a modification of the submission. Since if anything data-wise about it changed, publish date would have to be updated.